.gitlab-ci.yml

```diff
@@ -33,18 +34,18 @@ after_script:
 env:
   stage: prepare
   tags: [go]
+  before_script:
+    - go mod download
   artifacts:
     paths: [build/env.sh]
-  before_script:
-    - dep ensure -v
   script: go run mage.go -v GenerateEnvFile
 
 checks:
   stage: test
   tags: [go]
   script:
     - |
-      go get -u \
+      GO111MODULE=off go get -u \
```

Comment:
Since the source code is under GOPATH and Go modules are turned on go get will update local go.mod and go.sum file with all indirect dependencies which are used for these tools. By addding GO111MODULE=off it will preserve current behaviour of installing  tools to global GOPATH/bin.